src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  padding: 0 17px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.width-normal {
+  width: 200px;
+}

```

Comment:
You should set some class for `body`, for example, page, and make `card `it's element `page__card`